README.md

```diff
@@ -10,43 +10,12 @@ Odds are good that you don't want to invoke Ignition directly. In fact, it isn't
 
 - Mailing list: [coreos@lists.fedoraproject.org](https://lists.fedoraproject.org/archives/list/coreos@lists.fedoraproject.org/)
 - IRC: #[fedora-coreos](irc://irc.freenode.org:6697/#fedora-coreos) on freenode.org
-- Bugs: [issues][issues]
+- Reporting bugs: [issues](https://github.com/coreos/ignition/issues/new/choose)
 
 ## Contributing
 
 See [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
 
 To help triage or fix bugs, see the current [Ignition issues](https://github.com/coreos/ignition/issues/).
 
-## Reporting Bugs
-
-- To report a bug, use the [bug tracker][issues]
-
-## Config Validation
-
-To validate a config for Ignition there are binaries for a cli tool called `ignition-validate` available [on the releases page][releases]. There is also an ignition-validate container: `quay.io/coreos/ignition-validate`.
-
-Example:
-```
-# This example uses podman, but docker can be used too
-podman run --pull=always --rm -i quay.io/coreos/ignition-validate:release - < myconfig.ign
-```
-
-## Branches
```

Comment:
This section and the next one should stay in the README but be excluded from docs/.  They're not needed as developer documentation because `spec2x` and `ignition-dracut` are no longer maintained.  They might be useful, however, for people trying to figure out the state of the repository.